### PR TITLE
Tickets fragment

### PR DESCRIPTION
**Pull request by markarasev**
Enables direct links to "Tickets" section on the main page.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -28,6 +28,5 @@
   <body>
     <div id="app"></div>
     <script type="module" src="/root.js"></script>
-    <script type="text/javascript" src="https://assets.yurplan.com/yurplan-v1/dist/widget.js" async defer></script>
```
Comment (iusildra) [inline index.html:31]: Oups 😅

File: src/main/scala/io/scala/views/IndexView.scala [hunk 2]
```diff
@@ -69,6 +69,7 @@ case object IndexView extends ReactiveView[IndexPage] {
   )
 
   lazy val tickets: Div = div(
+    htmlAttr("id", StringAsIsCodec) := "tickets", // no built-in "id" attribute??
```
Comment (iusildra) [inline src/main/scala/io/scala/views/IndexView.scala:72]: Yes there is, it should be `idAttr := "tickets"`

Comment (markarasev) [reply src/main/scala/io/scala/views/IndexView.scala]: J'ai passé 10 minutes à le chercher... fixup : dc2ee2c3d24d0c3b3edcdb25b62e548da823b45a

**Review by iusildra — APPROVED** [review]
I'll look at the auto-deploy failure

**Pull request merged**